lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -45,9 +57,98 @@
     seed_milestone_relationship_types,
 )
 
+from lms.djangoapps.ccx.models import CustomCourseForEdX
+
 # pylint: disable=protected-access
 
 
+class CoachAccessTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
+    """
+    Test if user is coach on ccx.
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Set up course for tests
+        """
+        super(CoachAccessTestCaseCCX, cls).setUpClass()
```

Comment:
For both this and `setUp()`, how are you handling the multiple inheritance? I don't know what will happen if both `SharedModuleStoreTestCase` and `LoginEnrollmentTestCase` define `setUpClass()` -- will they both be called, or only one? (Even if only one of them defines `setUpClass()` right now, the other could do so in the future.)